README.md

```diff
@@ -13,9 +13,13 @@ The Kubeflow pipelines service has the following goals:
 * Easy experimentation: making it easy for you to try numerous ideas and techniques, and manage your various trials/experiments.
 * Easy re-use: enabling you to re-use components and pipelines to quickly cobble together end to end solutions, without having to re-build each time.
 
-## Documentation
+## Installation
+
+* Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
 
-Install Kubeflow Pipelines from [an overview of several options](https://www.kubeflow.org/docs/pipelines/installation/overview/).
+* :star: [Alpha] Starting from Kubeflow Pipelines 1.7, try out [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor) for not restricted by container runtime, and more secure pod access. (Docker runtime is no longer default option since Kubernetes 1.19)
```

Comment:
For discussion, I might prefer terms like:

* :star: [Alpha] Starting from Kubeflow Pipelines 1.7, try out [Emissary Executor](https://www.kubeflow.org/docs/components/pipelines/installation/choose-executor/#emissary-executor). Emissary executor is container runtime agnostic. The default docker executor depends on docker container runtime, which will be deprecated on Kubernetes 1.20+.

Reply:
Sounds good, updated the document with suggested term.